### PR TITLE
add 'I am the creator' checkbox to entry form

### PR DESCRIPTION
**Pull request by Pomax**
@mmmavis I'm not too sure about where the CSS should go here, or whether you already had a predefined class to make the checkbox look good.

Work left: figure out how to update formbuilder such that we can put this field "in" the label for the creators. According to the comps this should go between the creators label and the creators fields, and formbuilder does not allow for that right now.

### CODE REVIEW

**Comment by mmmavis** [comment]
related to https://github.com/mozilla/network-pulse/issues/573

**Comment by mmmavis** [comment]
@Pomax  I think first we need to move the checkbox after "Who are the creator...." . Not sure if this can be done without updating react-formbuilder 👀 

screen shot 2017-10-04 at 3 50 51 pm

**Comment by xmatthewx** [comment]
As an interim solution or to avoid complexity, I think we can do:

## Optional Details

Are you one of the creators?
 - [ ] I am one of the creators

Name any other creators. This could be staff, contributors, partners...

cc @kristinashu for sanity

**Comment by Pomax** [comment]
I've extended the formbuilder to allow for field guide texts, so that form now looks like this:

screen shot 2017-10-05 at 12 50 56 pm

**Comment by mmmavis** [comment]
Hey Pomax, I think for now we can just set `margin-bottom: 0` to `.published_by_creator .guide-text`. Once this is on staging we can ask @kristinashu to provide design suggestion for the form holistically.

File: scss/react-formbuilder.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+input.published-by-creator {
```
[the remark this answers is not in the document]
Comment (Pomax) [reply scss/react-formbuilder.scss]: missed that file, I shall relocate the code!

**Comment by mmmavis** [comment]
R+ Merging it now!

**Pull request merged**